src/index.ts

```diff
@@ -46,8 +47,33 @@ if (runningLocally) {
 }
 const firestore = firestoreAdmin.firestore();
 
+// setup discord.js commands
+client.on('ready', () => {
+  for (const [guildId] of client.guilds.cache) {
+    if (!client.user) {
+      console.error(client.user);
+      continue;
+    }
+
+    deployCommands(client.user.id, guildId, process.env.TOKEN!)
+  }
+});
+
+// listen for discord.js command
+client.on('interactionCreate', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	try {
+    const commandFile = require(`./commands/${interaction.commandName}`); // Loads up the command based on file name
+    await commandFile.execute(interaction, { client, firestore }); // Executes any function titled 'run' within the file
+	} catch (error) {
+		console.error(error);
+		await interaction.reply({ content: 'There was an error while executing this command!', ephemeral: true });
```

Comment:
Let's use a pattern of throwing strings on errors, and outputting those thrown strings to the user in this reply. Accessing the console to look for errors isn't an easy task with our current GCP configuration.